### DISCUSSION TITLE
Hi all! I'm Emma from the Course 1 videos 👋

### DISCUSSION THREAD

**Discussion by em-blue** [OpenMined/courses#84] [Introduce Yourself]
Hi everyone! Thanks for being interested in our courses, I really hope you like them! 👋

I'll share a bit about my own path here so you can see what led me to be interested in this topic! 😄 

## Why does a medical imaging person care about data privacy so much? 

After spending 5 years in medical imaging research, I've always been affected by data quality and quantity. I've experienced being data-rich ([working with the UK biobank data](https://www.sciencedirect.com/science/article/pii/S1053811920308867)) and data-poor ([working with clinical trial data](https://www.sciencedirect.com/science/article/pii/S2405630820300148) that resulted in some groups being as small as n=1 - yikes!). Because of this, I've always been really supportive of the open-data movement for research - I've seen the huge effect (for better or worse) data has had on my own research projects.

But in 2018, I tried my first real deep learning project on the UK biobank. The task was to predict age from the brain MRI to learn biomarkers of advanced aging. The project was successful, but beyond that, **I learned two important lessons that have changed my path completely:**

1. **Wow, data set size made a huge difference**. The moment I scaled up from training on 300 subjects (a pretty large sample size for imaging!) to 3000, my model immediately worked. In the end, I trained on around 7000, which achieved brilliant results. I didn't change my network... just the dataset. And I was just using a standard CNN, nothing fancy.
2. **Wow, we can tease information from data that we didn't know was in there.** My neural network was guessing the age of the subjects accurately. Maybe scans aren't so 'anonymized' after all - what else can neural networks infer from data? 

This realization started my interest in data privacy and wondering what else can be predicted from research data. **With horror, I learned about [re-facing medical images with GANs](https://arxiv.org/abs/1810.06455), [DNA phenotyping](https://blog.openmined.org/genetic-data-privacy/), and [identification via heartbeats](https://www.technologyreview.com/2019/06/27/238884/the-pentagon-has-a-laser-that-can-identify-people-from-a-distanceby-their-heartbeat/).**

At this point in my PhD, I had moved on to focus on a project focused on testing if a specific MRI technique is useful for measuring changes in oxygenation in tumor tissue. I inherited two glorious datasets from two clinical trials that had just finished, excited to analyze them and report on the insights. As I worked through the common data issues with clinical data (images flipped around, inconsistent naming and file structures, image misalignment, patient motion, image artefacts) **my resulting high-quality dataset size began to shrink**. In one study, one statistical "group" ended up as small as n=1. **Obviously, I can't conclude anything about the predictive nature of our technique if I only have n=1 in a group, which makes all of those years of research... all that work... inconclusive! And sadly this is just how a lot of research suffers from small datasets.**

Due to this mix of experience with data affecting my research, I am a bitter realist yet also very optimistic. My research has been a victim of our current data-sharing practices, yet I've also witnessed the incredible potential when high-quality data is available. **The tools we talk about in this course could make a serious difference for science - especially for transparency and reproducibility in science.** 

## This course is about hope

The more I learned about the data privacy situation at large, the more shocked I became. For me, learning about the reality of the data market ([like data brokers](https://www.webfx.com/blog/internet/what-are-data-brokers-and-what-is-your-data-worth-infographic/)) both horrified and motivated me. But - when I'd read all the scary stats to my partner, he'd explain that it all just made him feel intimidated - **it made him feel that data privacy today is a hopeless uphill battle that we (as the consumer) will never, ever win**. And I agree, a lot of writing about privacy today feels pretty hopeless.

**So in this course, we've taken a different approach.** 

There has already been enough great journalism on the scary stuff about privacy - read Privacy is Power (by Carissa Veliz) if you want more of that, she covers it well. _**But in this course, you'll notice that we don't spend time scaring you into caring about privacy.**_ In fact, there's really only one video ("Data is Fire") that talks about caring about data privacy for "privacy's sake". And we're also not just proposing that strict regulation will come and change everything.

**Instead, as you'll find out, the course is about all of the other reasons our data economy can (and will) change for the better** - not because of altruism alone, but because these new tools can help align the market incentives to a situation that _ends up being more privacy-preserving_. 

So - I've approached the writing of this course as a researcher, a consumer, and a community member - and I really do think we can have a better data situation in the near future. I'm full of hope for this, and by the end of the course, I hope you are too. 🤞

**Enjoy the course, and welcome to OpenMined!**

**Comment by iamtrask** [comment]
👋 Hi Emma!
